egs/aishell/ASR/local/compute_fbank_aishell.py

```diff
@@ -42,7 +49,9 @@
 torch.set_num_interop_threads(1)
 
 
-def compute_fbank_aishell(num_mel_bins: int = 80, perturb_speed: bool = False):
+def compute_fbank_aishell(
+    num_mel_bins: int = 80, perturb_speed: bool = False, whisper_fbank: bool = False
+):
     src_dir = Path("data/manifests")
     output_dir = Path("data/fbank")
```

Comment:
i'm wondering if we should use a different fbank dir if the feature is for whisper finetuning, like `data/fbank_whisper` and `data/fbank_whisper_v3` considering the difference among various versions. otherwise the original fbank might be overwritten.

Reply:
Added the options.